lib/Doctrine/Common/Collections/ArrayCollection.php

```diff
@@ -43,7 +43,7 @@ class ArrayCollection implements Collection, Selectable
      *
      * @var array
      */
-    private $elements;
+    private $elements = [];
```

Comment:
Why is initialization needed? This seems to always be an array as that's what is required in the constructor

Reply:
See #155 :)